src/trades_call_builder.js

```diff
@@ -0,0 +1,50 @@
+import {CallBuilder} from "./call_builder";
+
+/**
+ * Creates a new {@link TradesCallBuilder} pointed to server defined by serverUrl.
+ *
+ * Do not create this object directly, use {@link Server#orderbook}.
+ * @see [Trades](https://www.stellar.org/developers/horizon/reference/trades.html)
+ * @param {string} serverUrl serverUrl Horizon server URL.
+ */
+export class TradesCallBuilder extends CallBuilder {
+    constructor(serverUrl) {
+        super(serverUrl);
+        this.url.segment('trades');
+    }
+
+    /**
+    * Filter trades for a specific asset pair (orderbook)
+    * @param {Asset} base asset
+    * @param {Asset} counter asset
+    * @returns {TradesCallBuilder}
+    */
+    forAssetPair(base, counter) {
+        if (!base.isNative()) {
+            this.url.addQuery("base_asset_type", base.getAssetType());
+            this.url.addQuery("base_asset_code", base.getCode());
+            this.url.addQuery("base_asset_issuer", base.getIssuer());
+        } else {
+            this.url.addQuery("base_asset_type", 'native');
+        }
+        if (!counter.isNative()) {
+            this.url.addQuery("counter_asset_type", counter.getAssetType());
+            this.url.addQuery("counter_asset_code", counter.getCode());
+            this.url.addQuery("counter_asset_issuer", counter.getIssuer());
+        } else {
+            this.url.addQuery("counter_asset_type", 'native');
+        }
```

Comment:
Not in a scope of this PR but I see this code a lot. Would be great to move it to a helper function.